### PR TITLE
Derive Default for WriterProperties

### PR DESCRIPTION
**Pull request by tustvold**
# Which issue does this PR close?

Closes #.

# Rationale for this change
 

Having to construct a builder just to create the default writer properties is unnecessarily cumbersome

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

File: parquet/src/file/properties.rs [hunk 1]
```diff
@@ -130,6 +130,12 @@ pub struct WriterProperties {
     sorting_columns: Option<Vec<SortingColumn>>,
 }
 
+impl Default for WriterProperties {
+    fn default() -> Self {
+        Self::builder().build()
+    }
+}
+
 impl WriterProperties {
```
Comment (alamb) [inline parquet/src/file/properties.rs:139]: Can I also request `WriterProperties::new()` that does the same thing?

I think `X::new()` I think is the canonical way to construct new objects in Rust and will be the easiest for new Rust users to discover

File: parquet/src/file/properties.rs [hunk 2]
```diff
@@ -27,7 +27,7 @@
 //! };
 //!
 //! // Create properties with default configuration.
-//! let props = WriterProperties::builder().build();
+//! let props = WriterProperties::default();
```
Comment (alamb) [inline parquet/src/file/properties.rs:30]:
```suggestion
//! let props = WriterProperties::new();
```

See suggestion below

**Review by alamb — APPROVED** [review]
Thank you @tustvold

File: parquet/src/file/properties.rs (hunk 1 above)
Comment (tustvold) [reply parquet/src/file/properties.rs:139]: I prefer default as it indicates the default properties, but have added a new constructor as well

Comment (alamb) [reply parquet/src/file/properties.rs:139]: Thank you -- I think having them both is the best UX for users. Thank you 🙏

**Pull request merged**